### PR TITLE
peer detach with default port 

### PR DESCRIPTION
**Pull request by Madhu-1**
peer detach with default port if the port is not provided in peer detach CLI command

Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

File: glustercli/cmd/peer.go [hunk 1]
```diff
@@ -127,3 +131,38 @@ var poolListCmd = &cobra.Command{
 		peerStatusHandler(cmd)
 	},
 }
+
+// getPeerID return peerId of host
+func getPeerID(host string) (string, error) {
+	// Get Peers list to find Peer ID
```
Comment (aravindavk) [inline glustercli/cmd/peer.go]: Add one more check to see the input is already peer id. (If Peer detach command is called by giving peer id itself)

    if uuid.Parse(host) != nil {
        return host, nil
    }

Comment (Madhu-1) [reply glustercli/cmd/peer.go]: do we need to compare that peerID with the available peer?
or let the server take care of it?

Comment (Madhu-1) [reply glustercli/cmd/peer.go]: added check

**Comment by Madhu-1** [comment]
@aravindavk @prashanthpai @kshlm  please review

**Comment by aravindavk** [comment]
retest this please

**Pull request merged**